cernopendata/modules/fixtures/data/articles/welcome/welcome.md

```diff
@@ -0,0 +1,13 @@
+FIXME PRELIMINARY TEXT
+
+With the growing demand of open science data and increased interest in the CERN Open Data portal, we were working hard on modernizing our web presence. Here, we are telling you more about the changes we made and how you can use the updated portal.
+
+ <img src="../../../../static/assets/header-image.svg" width="70%" align="middle">
```

Comment:
Does this render for you?

![screen shot 2017-11-17 at 10 44 26 am](https://user-images.githubusercontent.com/9035606/32940899-4ea2f04e-cb84-11e7-8e42-1d5752054c00.png)

Most probably it is related with https://stackoverflow.com/questions/32123484/svg-use-external-source-not-working-on-firefox/32124010#32124010 (discovered by @hjhsalo) you could either try that fix or convert the `svg` to `png`

Reply:
Also the dots should be removed and the link should be ```/static/.../```.

Reply:
now fixed :+1: